app/validation/validators.py

```diff
@@ -71,11 +72,35 @@ def __init__(self, minimum=None, maximum=None, messages=None):
 
     def __call__(self, form, field):
         data = field.data
+        error_message = None
         if data is not None:
             if self.minimum is not None and data < self.minimum:
-                raise validators.ValidationError(self.messages['NEGATIVE_INTEGER'])
+                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_SMALL'])
             elif self.maximum is not None and data > self.maximum:
-                raise validators.ValidationError(self.messages['INTEGER_TOO_LARGE'])
+                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_LARGE'])
```

Comment:
Shouldn't 'OUT_OF_RANGE' be the second arg here?
As in `self.messages.get('NUMBER_TOO_LARGE', self.messages['OUT_OF_RANGE'])`